Flaky HttpServerTest.testHttpServerBind test

### To reproduce

```
2025-09-22T18:00:10.3708094Z 2025-09-22T18:00:10.175164Z I i.q.t.TestListener >>>> io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind
2025-09-22T18:00:10.3709693Z 2025-09-22T18:00:10.175418Z I i.q.t.AbstractTest Starting test HttpServerTest#testHttpServerBind
2025-09-22T18:00:10.3711784Z 2025-09-22T18:00:10.175611Z I i.q.c.t.t.InputFormatConfiguration loading input format config [resource=/text_loader.json]
2025-09-22T18:00:10.3713611Z 2025-09-22T18:00:10.176415Z A IODispatcher listening on 0.0.0.0:9001 [fd=100932691861 backlog=64]
2025-09-22T18:00:10.3715246Z 2025-09-22T18:00:10.176526Z I i.q.t.c.h.HttpServerTest$HttpServerMock worker pool started [pool=testing]
2025-09-22T18:00:11.1632889Z Thread dump: 
2025-09-22T18:00:11.1633859Z 'main': RUNNABLE
2025-09-22T18:00:11.1634318Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollAccessor.epollWait(Native Method)
2025-09-22T18:00:11.1634778Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollAccessor.epollWait(EpollAccessor.java:52)
2025-09-22T18:00:11.1636740Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollFacadeImpl.epollWait(EpollFacadeImpl.java:46)
2025-09-22T18:00:11.1637794Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.Epoll.poll(Epoll.java:91)
2025-09-22T18:00:11.1640005Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClientLinux.ioWait(HttpClientLinux.java:58)
2025-09-22T18:00:11.1641795Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:187)
2025-09-22T18:00:11.1643592Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:194)
2025-09-22T18:00:11.1645609Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:832)
2025-09-22T18:00:11.1647674Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:825)
2025-09-22T18:00:11.1649517Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.assertHttpRequest(HttpServerTest.java:114)
2025-09-22T18:00:11.1651478Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.assertExecRequest(HttpServerTest.java:103)
2025-09-22T18:00:11.1653350Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind(HttpServerTest.java:89)
2025-09-22T18:00:11.1655024Z 		at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2025-09-22T18:00:11.1656907Z 		at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2025-09-22T18:00:11.1658915Z 		at java.base@17.0.10/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2025-09-22T18:00:11.1660152Z 		at java.base@17.0.10/java.lang.reflect.Method.invoke(Method.java:568)
2025-09-22T18:00:11.1661992Z 		at app/junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2025-09-22T18:00:11.1663774Z 		at app/junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2025-09-22T18:00:11.1665582Z 		at app/junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2025-09-22T18:00:11.1667339Z 		at app/junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2025-09-22T18:00:11.1667678Z 
2025-09-22T18:00:11.1668102Z 
2025-09-22T18:00:11.1668688Z 'Reference Handler': RUNNABLE
2025-09-22T18:00:11.1669398Z 		at java.base@17.0.10/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
2025-09-22T18:00:11.1671014Z 		at java.base@17.0.10/java.lang.ref.Reference.processPendingReferences(Reference.java:253)
2025-09-22T18:00:11.1672500Z 		at java.base@17.0.10/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)
2025-09-22T18:00:11.1672844Z 
2025-09-22T18:00:11.1673274Z 
2025-09-22T18:00:11.1673834Z 'Finalizer': WAITING
2025-09-22T18:00:11.1674244Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.1675278Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.1676756Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
2025-09-22T18:00:11.1679736Z 		at java.base@17.0.10/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:172)
2025-09-22T18:00:11.1680066Z 
2025-09-22T18:00:11.1680729Z 
2025-09-22T18:00:11.1681320Z 'Signal Dispatcher': RUNNABLE
2025-09-22T18:00:11.1681657Z 
2025-09-22T18:00:11.1682087Z 
2025-09-22T18:00:11.1682828Z 'Common-Cleaner': TIMED_WAITING
2025-09-22T18:00:11.1683267Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.1683754Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.1684251Z 		at java.base@17.0.10/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
2025-09-22T18:00:11.1684730Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1685705Z 		at java.base@17.0.10/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
2025-09-22T18:00:11.1686090Z 
2025-09-22T18:00:11.1686514Z 
2025-09-22T18:00:11.1687106Z 'Notification Thread': RUNNABLE
2025-09-22T18:00:11.1687420Z 
2025-09-22T18:00:11.1687848Z 
2025-09-22T18:00:11.1688408Z 'pool-1-thread-1': RUNNABLE
2025-09-22T18:00:11.1688810Z 		at java.base@17.0.10/sun.nio.ch.EPoll.wait(Native Method)
2025-09-22T18:00:11.1689372Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.poll(EPollPort.java:200)
2025-09-22T18:00:11.1690756Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.run(EPollPort.java:281)
2025-09-22T18:00:11.1692653Z 		at java.base@17.0.10/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
2025-09-22T18:00:11.1694419Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2025-09-22T18:00:11.1696171Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.1697235Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1697605Z 
2025-09-22T18:00:11.1698039Z 
2025-09-22T18:00:11.1698604Z 'pool-1-thread-2': WAITING
2025-09-22T18:00:11.1699019Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.1700439Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
2025-09-22T18:00:11.1702745Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
2025-09-22T18:00:11.1704504Z 		at java.base@17.0.10/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3465)
2025-09-22T18:00:11.1706059Z 		at java.base@17.0.10/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3436)
2025-09-22T18:00:11.1708415Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
2025-09-22T18:00:11.1710053Z 		at java.base@17.0.10/java.util.concurrent.ArrayBlockingQueue.take(ArrayBlockingQueue.java:420)
2025-09-22T18:00:11.1711482Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.run(EPollPort.java:275)
2025-09-22T18:00:11.1713364Z 		at java.base@17.0.10/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
2025-09-22T18:00:11.1715083Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2025-09-22T18:00:11.1716816Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.1717855Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1718223Z 
2025-09-22T18:00:11.1718661Z 
2025-09-22T18:00:11.1719262Z 'surefire-forkedjvm-stream-flusher': TIMED_WAITING
2025-09-22T18:00:11.1719917Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.1721555Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
2025-09-22T18:00:11.1724005Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
2025-09-22T18:00:11.1726263Z 		at java.base@17.0.10/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
2025-09-22T18:00:11.1728557Z 		at java.base@17.0.10/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
2025-09-22T18:00:11.1730239Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
2025-09-22T18:00:11.1731990Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
2025-09-22T18:00:11.1733840Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.1734843Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1735169Z 
2025-09-22T18:00:11.1735768Z 
2025-09-22T18:00:11.1736379Z 'surefire-forkedjvm-command-thread': WAITING
2025-09-22T18:00:11.1736821Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.1738325Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
2025-09-22T18:00:11.1740396Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquire(AbstractQueuedSynchronizer.java:715)
2025-09-22T18:00:11.1742888Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1047)
2025-09-22T18:00:11.1744325Z 		at java.base@17.0.10/java.util.concurrent.CountDownLatch.await(CountDownLatch.java:230)
2025-09-22T18:00:11.1745582Z 		at java.base@17.0.10/sun.nio.ch.PendingFuture.get(PendingFuture.java:178)
2025-09-22T18:00:11.1747178Z 		at app//org.apache.maven.surefire.api.util.internal.Channels$2.read(Channels.java:127)
2025-09-22T18:00:11.1748591Z 		at java.base@17.0.10/java.io.BufferedInputStream.fill(BufferedInputStream.java:244)
2025-09-22T18:00:11.1750066Z 		at java.base@17.0.10/java.io.BufferedInputStream.read1(BufferedInputStream.java:284)
2025-09-22T18:00:11.1751517Z 		at java.base@17.0.10/java.io.BufferedInputStream.read(BufferedInputStream.java:343)
2025-09-22T18:00:11.1753157Z 		at app//org.apache.maven.surefire.api.util.internal.Channels$3.readImpl(Channels.java:169)
2025-09-22T18:00:11.1756985Z 		at app//org.apache.maven.surefire.api.util.internal.AbstractNoninterruptibleReadableChannel.read(AbstractNoninterruptibleReadableChannel.java:50)
2025-09-22T18:00:11.1757424Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.read(AbstractStreamDecoder.java:430)
2025-09-22T18:00:11.1759339Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.read(AbstractStreamDecoder.java:419)
2025-09-22T18:00:11.1761333Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.readMessageType(AbstractStreamDecoder.java:116)
2025-09-22T18:00:11.1762923Z 		at app//org.apache.maven.surefire.booter.stream.CommandDecoder.decode(CommandDecoder.java:77)
2025-09-22T18:00:11.1764779Z 		at app//org.apache.maven.surefire.booter.spi.CommandChannelDecoder.decode(CommandChannelDecoder.java:60)
2025-09-22T18:00:11.1766547Z 		at app//org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:290)
2025-09-22T18:00:11.1767756Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1768130Z 
2025-09-22T18:00:11.1768567Z 
2025-09-22T18:00:11.1769142Z 'logging_0': TIMED_WAITING
2025-09-22T18:00:11.1769590Z 		at java.base@17.0.10/java.lang.Thread.sleep(Native Method)
2025-09-22T18:00:11.1770415Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.std.Os.sleep(Os.java:233)
2025-09-22T18:00:11.1771765Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.mp.Worker.run(Worker.java:184)
2025-09-22T18:00:11.1771967Z 
2025-09-22T18:00:11.1772362Z 
2025-09-22T18:00:11.1772912Z 'Thread-2': RUNNABLE
2025-09-22T18:00:11.1773605Z 		at java.management@17.0.10/sun.management.ThreadImpl.getThreadInfo1(Native Method)
2025-09-22T18:00:11.1775125Z 		at java.management@17.0.10/sun.management.ThreadImpl.getThreadInfo(ThreadImpl.java:197)
2025-09-22T18:00:11.1776656Z 		at app/io.questdb.test/io.questdb.test.TestListener.dumpThreadStacks(TestListener.java:48)
2025-09-22T18:00:11.1778355Z 		at app/io.questdb.test/io.questdb.test.TestListener.lambda$static$0(TestListener.java:116)
2025-09-22T18:00:11.1780020Z 		at app/io.questdb.test/io.questdb.test.TestListener$$Lambda$146/0x00007f67681fcf40.run(Unknown Source)
2025-09-22T18:00:11.1781053Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1781427Z 
2025-09-22T18:00:11.1781859Z 
2025-09-22T18:00:11.1782430Z 'Thread-3': TIMED_WAITING
2025-09-22T18:00:11.1782900Z 		at java.base@17.0.10/java.lang.Thread.sleep(Native Method)
2025-09-22T18:00:11.1783870Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.std.Os.sleep(Os.java:233)
2025-09-22T18:00:11.1785381Z 		at app/io.questdb.test/io.questdb.test.TestListener.lambda$static$1(TestListener.java:133)
2025-09-22T18:00:11.1787112Z 		at app/io.questdb.test/io.questdb.test.TestListener$$Lambda$147/0x00007f67681fd160.run(Unknown Source)
2025-09-22T18:00:11.1788187Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1788635Z 
2025-09-22T18:00:11.1788820Z 
2025-09-22T18:00:11.1789355Z 'PostgreSQL-JDBC-Cleaner': TIMED_WAITING
2025-09-22T18:00:11.1789898Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.1791359Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.1792974Z 		at app/org.postgresql.jdbc@42.7.4/org.postgresql.util.LazyCleaner$1.run(LazyCleaner.java:129)
2025-09-22T18:00:11.1794224Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.1794512Z 
2025-09-22T18:00:11.1794587Z 
2025-09-22T18:00:11.3032941Z 2025-09-22T18:00:11.302035Z A dump-thread-stacks 
2025-09-22T18:00:11.3034283Z 'main': RUNNABLE
2025-09-22T18:00:11.3035087Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollAccessor.epollWait(Native Method)
2025-09-22T18:00:11.3035957Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollAccessor.epollWait(EpollAccessor.java:52)
2025-09-22T18:00:11.3037118Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.EpollFacadeImpl.epollWait(EpollFacadeImpl.java:46)
2025-09-22T18:00:11.3038428Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.network.Epoll.poll(Epoll.java:91)
2025-09-22T18:00:11.3040420Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClientLinux.ioWait(HttpClientLinux.java:58)
2025-09-22T18:00:11.3042279Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:187)
2025-09-22T18:00:11.3044110Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:194)
2025-09-22T18:00:11.3046147Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:832)
2025-09-22T18:00:11.3048188Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:825)
2025-09-22T18:00:11.3050243Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.assertHttpRequest(HttpServerTest.java:114)
2025-09-22T18:00:11.3051919Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.assertExecRequest(HttpServerTest.java:103)
2025-09-22T18:00:11.3053981Z 		at app/io.questdb.test/io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind(HttpServerTest.java:89)
2025-09-22T18:00:11.3055477Z 		at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2025-09-22T18:00:11.3057384Z 		at java.base@17.0.10/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2025-09-22T18:00:11.3059384Z 		at java.base@17.0.10/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2025-09-22T18:00:11.3060923Z 		at java.base@17.0.10/java.lang.reflect.Method.invoke(Method.java:568)
2025-09-22T18:00:11.3062477Z 		at app/junit@4.13.2/org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2025-09-22T18:00:11.3064321Z 		at app/junit@4.13.2/org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2025-09-22T18:00:11.3066089Z 		at app/junit@4.13.2/org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2025-09-22T18:00:11.3067850Z 		at app/junit@4.13.2/org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2025-09-22T18:00:11.3068183Z 
2025-09-22T18:00:11.3068665Z 
2025-09-22T18:00:11.3069085Z 2025-09-22T18:00:11.302069Z A dump-thread-stacks 
2025-09-22T18:00:11.3069718Z 'Reference Handler': RUNNABLE
2025-09-22T18:00:11.3071094Z 		at java.base@17.0.10/java.lang.ref.Reference.waitForReferencePendingList(Native Method)
2025-09-22T18:00:11.3072421Z 		at java.base@17.0.10/java.lang.ref.Reference.processPendingReferences(Reference.java:253)
2025-09-22T18:00:11.3073906Z 		at java.base@17.0.10/java.lang.ref.Reference$ReferenceHandler.run(Reference.java:215)
2025-09-22T18:00:11.3074204Z 
2025-09-22T18:00:11.3074697Z 
2025-09-22T18:00:11.3075099Z 2025-09-22T18:00:11.302077Z A dump-thread-stacks 
2025-09-22T18:00:11.3075711Z 'Finalizer': WAITING
2025-09-22T18:00:11.3076376Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.3077758Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.3081765Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:176)
2025-09-22T18:00:11.3082242Z 		at java.base@17.0.10/java.lang.ref.Finalizer$FinalizerThread.run(Finalizer.java:172)
2025-09-22T18:00:11.3082614Z 
2025-09-22T18:00:11.3083048Z 
2025-09-22T18:00:11.3083678Z 2025-09-22T18:00:11.302082Z A dump-thread-stacks 
2025-09-22T18:00:11.3084102Z 'Signal Dispatcher': RUNNABLE
2025-09-22T18:00:11.3084440Z 
2025-09-22T18:00:11.3084879Z 
2025-09-22T18:00:11.3085512Z 2025-09-22T18:00:11.302083Z A dump-thread-stacks 
2025-09-22T18:00:11.3085960Z 'Common-Cleaner': TIMED_WAITING
2025-09-22T18:00:11.3086384Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.3103973Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.3104408Z 		at java.base@17.0.10/jdk.internal.ref.CleanerImpl.run(CleanerImpl.java:140)
2025-09-22T18:00:11.3104963Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3105480Z 		at java.base@17.0.10/jdk.internal.misc.InnocuousThread.run(InnocuousThread.java:162)
2025-09-22T18:00:11.3105827Z 
2025-09-22T18:00:11.3106255Z 
2025-09-22T18:00:11.3106848Z 2025-09-22T18:00:11.302089Z A dump-thread-stacks 
2025-09-22T18:00:11.3107234Z 'Notification Thread': RUNNABLE
2025-09-22T18:00:11.3107573Z 
2025-09-22T18:00:11.3108005Z 
2025-09-22T18:00:11.3108612Z 2025-09-22T18:00:11.302090Z A dump-thread-stacks 
2025-09-22T18:00:11.3109035Z 'pool-1-thread-1': RUNNABLE
2025-09-22T18:00:11.3109481Z 		at java.base@17.0.10/sun.nio.ch.EPoll.wait(Native Method)
2025-09-22T18:00:11.3109967Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.poll(EPollPort.java:200)
2025-09-22T18:00:11.3110452Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.run(EPollPort.java:281)
2025-09-22T18:00:11.3110935Z 		at java.base@17.0.10/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
2025-09-22T18:00:11.3111402Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2025-09-22T18:00:11.3111886Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.3112351Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3112678Z 
2025-09-22T18:00:11.3113101Z 
2025-09-22T18:00:11.3113704Z 2025-09-22T18:00:11.302096Z A dump-thread-stacks 
2025-09-22T18:00:11.3114148Z 'pool-1-thread-2': WAITING
2025-09-22T18:00:11.3114558Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.3115095Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.park(LockSupport.java:341)
2025-09-22T18:00:11.3115593Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionNode.block(AbstractQueuedSynchronizer.java:506)
2025-09-22T18:00:11.3116078Z 		at java.base@17.0.10/java.util.concurrent.ForkJoinPool.unmanagedBlock(ForkJoinPool.java:3465)
2025-09-22T18:00:11.3116558Z 		at java.base@17.0.10/java.util.concurrent.ForkJoinPool.managedBlock(ForkJoinPool.java:3436)
2025-09-22T18:00:11.3117095Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.await(AbstractQueuedSynchronizer.java:1623)
2025-09-22T18:00:11.3117766Z 		at java.base@17.0.10/java.util.concurrent.ArrayBlockingQueue.take(ArrayBlockingQueue.java:420)
2025-09-22T18:00:11.3119113Z 		at java.base@17.0.10/sun.nio.ch.EPollPort$EventHandlerTask.run(EPollPort.java:275)
2025-09-22T18:00:11.3121007Z 		at java.base@17.0.10/sun.nio.ch.AsynchronousChannelGroupImpl$1.run(AsynchronousChannelGroupImpl.java:113)
2025-09-22T18:00:11.3122922Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
2025-09-22T18:00:11.3124522Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.3125547Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3125907Z 
2025-09-22T18:00:11.3126354Z 
2025-09-22T18:00:11.3126960Z 2025-09-22T18:00:11.302106Z A dump-thread-stacks 
2025-09-22T18:00:11.3127376Z 'surefire-forkedjvm-stream-flusher': TIMED_WAITING
2025-09-22T18:00:11.3128507Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.3130378Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:252)
2025-09-22T18:00:11.3132602Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:1672)
2025-09-22T18:00:11.3134991Z 		at java.base@17.0.10/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
2025-09-22T18:00:11.3137229Z 		at java.base@17.0.10/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
2025-09-22T18:00:11.3138912Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1062)
2025-09-22T18:00:11.3140720Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1122)
2025-09-22T18:00:11.3142456Z 		at java.base@17.0.10/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
2025-09-22T18:00:11.3143477Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3143828Z 
2025-09-22T18:00:11.3144265Z 
2025-09-22T18:00:11.3144856Z 2025-09-22T18:00:11.302114Z A dump-thread-stacks 
2025-09-22T18:00:11.3145343Z 'surefire-forkedjvm-command-thread': WAITING
2025-09-22T18:00:11.3146332Z 		at java.base@17.0.10/jdk.internal.misc.Unsafe.park(Native Method)
2025-09-22T18:00:11.3147845Z 		at java.base@17.0.10/java.util.concurrent.locks.LockSupport.park(LockSupport.java:211)
2025-09-22T18:00:11.3149919Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquire(AbstractQueuedSynchronizer.java:715)
2025-09-22T18:00:11.3152415Z 		at java.base@17.0.10/java.util.concurrent.locks.AbstractQueuedSynchronizer.acquireSharedInterruptibly(AbstractQueuedSynchronizer.java:1047)
2025-09-22T18:00:11.3153870Z 		at java.base@17.0.10/java.util.concurrent.CountDownLatch.await(CountDownLatch.java:230)
2025-09-22T18:00:11.3155209Z 		at java.base@17.0.10/sun.nio.ch.PendingFuture.get(PendingFuture.java:178)
2025-09-22T18:00:11.3156707Z 		at app//org.apache.maven.surefire.api.util.internal.Channels$2.read(Channels.java:127)
2025-09-22T18:00:11.3158817Z 		at java.base@17.0.10/java.io.BufferedInputStream.fill(BufferedInputStream.java:244)
2025-09-22T18:00:11.3159723Z 		at java.base@17.0.10/java.io.BufferedInputStream.read1(BufferedInputStream.java:284)
2025-09-22T18:00:11.3161178Z 		at java.base@17.0.10/java.io.BufferedInputStream.read(BufferedInputStream.java:343)
2025-09-22T18:00:11.3162832Z 		at app//org.apache.maven.surefire.api.util.internal.Channels$3.readImpl(Channels.java:169)
2025-09-22T18:00:11.3165346Z 		at app//org.apache.maven.surefire.api.util.internal.AbstractNoninterruptibleReadableChannel.read(AbstractNoninterruptibleReadableChannel.java:50)
2025-09-22T18:00:11.3167180Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.read(AbstractStreamDecoder.java:430)
2025-09-22T18:00:11.3168984Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.read(AbstractStreamDecoder.java:419)
2025-09-22T18:00:11.3171004Z 		at app//org.apache.maven.surefire.api.stream.AbstractStreamDecoder.readMessageType(AbstractStreamDecoder.java:116)
2025-09-22T18:00:11.3172691Z 		at app//org.apache.maven.surefire.booter.stream.CommandDecoder.decode(CommandDecoder.java:77)
2025-09-22T18:00:11.3174685Z 		at app//org.apache.maven.surefire.booter.spi.CommandChannelDecoder.decode(CommandChannelDecoder.java:60)
2025-09-22T18:00:11.3176445Z 		at app//org.apache.maven.surefire.booter.CommandReader$CommandRunnable.run(CommandReader.java:290)
2025-09-22T18:00:11.3177486Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3177849Z 
2025-09-22T18:00:11.3178269Z 
2025-09-22T18:00:11.3178881Z 2025-09-22T18:00:11.302131Z A dump-thread-stacks 
2025-09-22T18:00:11.3179330Z 'logging_0': TIMED_WAITING
2025-09-22T18:00:11.3179880Z 		at java.base@17.0.10/java.lang.Thread.sleep(Native Method)
2025-09-22T18:00:11.3181158Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.std.Os.sleep(Os.java:233)
2025-09-22T18:00:11.3182545Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.mp.Worker.run(Worker.java:184)
2025-09-22T18:00:11.3182871Z 
2025-09-22T18:00:11.3183296Z 
2025-09-22T18:00:11.3183900Z 2025-09-22T18:00:11.302134Z A dump-thread-stacks 
2025-09-22T18:00:11.3184316Z 'Thread-2': TIMED_WAITING
2025-09-22T18:00:11.3184927Z 		at java.base@17.0.10/java.lang.Thread.sleep(Native Method)
2025-09-22T18:00:11.3186315Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.std.Os.sleep(Os.java:233)
2025-09-22T18:00:11.3187746Z 		at app/io.questdb.test/io.questdb.test.TestListener.lambda$static$0(TestListener.java:117)
2025-09-22T18:00:11.3189557Z 		at app/io.questdb.test/io.questdb.test.TestListener$$Lambda$146/0x00007f67681fcf40.run(Unknown Source)
2025-09-22T18:00:11.3190535Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3190922Z 
2025-09-22T18:00:11.3191363Z 
2025-09-22T18:00:11.3191960Z 2025-09-22T18:00:11.302139Z A dump-thread-stacks 
2025-09-22T18:00:11.3192460Z 'Thread-3': RUNNABLE
2025-09-22T18:00:11.3193316Z 		at java.management@17.0.10/sun.management.ThreadImpl.getThreadInfo1(Native Method)
2025-09-22T18:00:11.3194798Z 		at java.management@17.0.10/sun.management.ThreadImpl.getThreadInfo(ThreadImpl.java:197)
2025-09-22T18:00:11.3197877Z 		at app/io.questdb@9.0.4-SNAPSHOT/io.questdb.griffin.engine.functions.catalogue.DumpThreadStacksFunctionFactory.dumpThreadStacks(DumpThreadStacksFunctionFactory.java:54)
2025-09-22T18:00:11.3199361Z 		at app/io.questdb.test/io.questdb.test.TestListener.lambda$static$1(TestListener.java:132)
2025-09-22T18:00:11.3201156Z 		at app/io.questdb.test/io.questdb.test.TestListener$$Lambda$147/0x00007f67681fd160.run(Unknown Source)
2025-09-22T18:00:11.3202192Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3202579Z 
2025-09-22T18:00:11.3203009Z 
2025-09-22T18:00:11.3203734Z 2025-09-22T18:00:11.302145Z A dump-thread-stacks 
2025-09-22T18:00:11.3204233Z 'PostgreSQL-JDBC-Cleaner': TIMED_WAITING
2025-09-22T18:00:11.3204993Z 		at java.base@17.0.10/java.lang.Object.wait(Native Method)
2025-09-22T18:00:11.3206401Z 		at java.base@17.0.10/java.lang.ref.ReferenceQueue.remove(ReferenceQueue.java:155)
2025-09-22T18:00:11.3208070Z 		at app/org.postgresql.jdbc@42.7.4/org.postgresql.util.LazyCleaner$1.run(LazyCleaner.java:129)
2025-09-22T18:00:11.3209175Z 		at java.base@17.0.10/java.lang.Thread.run(Thread.java:840)
2025-09-22T18:00:11.3209505Z 
2025-09-22T18:00:11.3209597Z 
2025-09-22T18:01:10.2339165Z 2025-09-22T18:01:10.230628Z I IODispatcher closed
2025-09-22T18:01:10.2340362Z 2025-09-22T18:01:10.230684Z I i.q.t.AbstractTest Finished test HttpServerTest#testHttpServerBind
2025-09-22T18:01:10.2340874Z 2025-09-22T18:01:10.232807Z E i.q.t.TestListener ***** Test Failed ***** io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind duration_ms=60057 : 
2025-09-22T18:01:10.2341194Z io.questdb.cutlass.http.client.HttpClientException: timed out [errno=11]
2025-09-22T18:01:10.2341477Z 	at io.questdb.cutlass.http.client.HttpClient.dieWaiting(HttpClient.java:223)
2025-09-22T18:01:10.2341815Z 	at io.questdb.cutlass.http.client.HttpClientLinux.ioWait(HttpClientLinux.java:58)
2025-09-22T18:01:10.2342128Z 	at io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:187)
2025-09-22T18:01:10.2343413Z 	at io.questdb.cutlass.http.client.HttpClient.recvOrDie(HttpClient.java:194)
2025-09-22T18:01:10.2344962Z 	at io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:832)
2025-09-22T18:01:10.2346436Z 	at io.questdb.cutlass.http.client.HttpClient$ResponseHeaders.await(HttpClient.java:825)
2025-09-22T18:01:10.2348080Z 	at io.questdb.test.cutlass.http.HttpServerTest.assertHttpRequest(HttpServerTest.java:114)
2025-09-22T18:01:10.2349530Z 	at io.questdb.test.cutlass.http.HttpServerTest.assertExecRequest(HttpServerTest.java:103)
2025-09-22T18:01:10.2351078Z 	at io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind(HttpServerTest.java:89)
2025-09-22T18:01:10.2352298Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2025-09-22T18:01:10.2353891Z 	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
2025-09-22T18:01:10.2355642Z 	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2025-09-22T18:01:10.2356501Z 	at java.lang.reflect.Method.invoke(Method.java:568)
2025-09-22T18:01:10.2358054Z 	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
2025-09-22T18:01:10.2359507Z 	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
2025-09-22T18:01:10.2361041Z 	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
2025-09-22T18:01:10.2362425Z 	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
2025-09-22T18:01:10.2363858Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2025-09-22T18:01:10.2365236Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2025-09-22T18:01:10.2375258Z 	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
2025-09-22T18:01:10.2375587Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2025-09-22T18:01:10.2375905Z 	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
2025-09-22T18:01:10.2376198Z 	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
2025-09-22T18:01:10.2376507Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
2025-09-22T18:01:10.2387937Z <<<<= io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind duration_ms=60057
2025-09-22T18:01:10.2396067Z 	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
2025-09-22T18:01:10.2396441Z 	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
2025-09-22T18:01:10.2396728Z 	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
2025-09-22T18:01:10.2402208Z 	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
2025-09-22T18:01:10.2402492Z 	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
2025-09-22T18:01:10.2402754Z 	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
2025-09-22T18:01:10.2403023Z 	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
2025-09-22T18:01:10.2408611Z 	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
2025-09-22T18:01:10.2408926Z 	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:54)
2025-09-22T18:01:10.2409325Z 	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
2025-09-22T18:01:10.2409582Z 	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
2025-09-22T18:01:10.2414760Z 	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
2025-09-22T18:01:10.2415045Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.execute(JUnit4Provider.java:316)
2025-09-22T18:01:10.2415338Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeWithRerun(JUnit4Provider.java:240)
2025-09-22T18:01:10.2421508Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.executeTestSet(JUnit4Provider.java:214)
2025-09-22T18:01:10.2421832Z 	at org.apache.maven.surefire.junit4.JUnit4Provider.invoke(JUnit4Provider.java:155)
2025-09-22T18:01:10.2422127Z 	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
2025-09-22T18:01:10.2422413Z 	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
2025-09-22T18:01:10.2422957Z 	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
2025-09-22T18:01:10.2423600Z 	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
2025-09-22T18:01:10.2423993Z 
2025-09-22T18:01:10.2425745Z 2025-09-22T18:01:10.232862Z I i.q.t.TestListener <<<< io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind duration_ms=60057
2025-09-22T18:01:10.2436167Z [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 60.06 s <<< FAILURE! -- in io.questdb.test.cutlass.http.HttpServerTest
2025-09-22T18:01:10.2450390Z [ERROR] io.questdb.test.cutlass.http.HttpServerTest.testHttpServerBind -- Time elapsed: 60.05 s <<< ERROR!
2025-09-22T18:01:10.2450768Z io.questdb.cutlass.http.client.HttpClientException: timed out [errno=11]

```

### QuestDB version:

9.0.3

### OS, in case of Docker specify Docker and the Host OS:

ubuntu

### File System, in case of Docker specify Host File System:

zfs

### Full Name:

Alex Pelagenko

### Affiliation:

QuestDB

### Have you followed Linux, MacOs kernel configuration steps to increase Maximum open files and Maximum virtual memory areas limit?

- [x] Yes, I have

### Additional context

_No response_

--- a/core/src/test/java/io/questdb/test/cutlass/http/HttpServerTest.java
+++ b/core/src/test/java/io/questdb/test/cutlass/http/HttpServerTest.java
@@ -59,6 +59,7 @@ import org.junit.Assert;
 import org.junit.Test;
 
 import static io.questdb.test.tools.TestUtils.assertEquals;
+import static java.lang.Math.max;
 import static java.lang.Math.min;
 import static org.junit.Assert.assertTrue;
 
@@ -151,7 +152,7 @@ public class HttpServerTest extends AbstractTest {
                     .withPort(port)
                     .build(new DefaultTestCairoConfiguration(root));
 
-            final int cpuAvailable = Runtime.getRuntime().availableProcessors() / 2;
+            final int cpuAvailable = max(Runtime.getRuntime().availableProcessors() / 2, 1);
             workerPool = new TestWorkerPool(min(workerCount, cpuAvailable));
             httpServer = new HttpServer(httpConfiguration, workerPool, PlainSocketFactory.INSTANCE);
         }

--- a/core/src/test/java/io/questdb/test/mp/TestWorkerPool.java
+++ b/core/src/test/java/io/questdb/test/mp/TestWorkerPool.java
@@ -63,6 +63,7 @@ public class TestWorkerPool extends WorkerPool {
                 return workerCount;
             }
         });
+        assert workerCount > 0;
     }
 
     public TestWorkerPool(WorkerPoolConfiguration configuration) {

